pkg/morph/client/static.go

```diff
@@ -97,6 +99,14 @@ func (i *InvokePrmOptional) SetHash(hash util.Uint256) {
 	i.hash = &hash
 }
 
+// NotaryAlphabetSignature makes client send notary request instead of a
+// regular signed transaction. Such a request should be received and signed by
+// the Alphabet, otherwise the plug (empty) transaction will be added to the
+// chain.
+func (ti *InvokePrmOptional) NotaryAlphabetSignature(signByAlphabet bool) {
```

Comment:
Can be `RequireAlphabetSignature()`, the default is `false`, this will just set it to `true`.

Reply:
dont mind, wanted it to be flexible, but that is just prm struct, not needed. fixed